.yarn/versions/fa5cc541.yml

```
@@ -0,0 +1,30 @@
+releases:
+  "@yarnpkg/cli": minor
```

```suggestion
  "@yarnpkg/cli": major
```

Since `--skip-builds` wont work anymore it's a breaking change